### ISSUE TITLE
Air conditioner 

### ISSUE THREAD

**Issue by Draghmar** [Galveston01/tuya-adapter#8]
Hi.
I have a air conditioner (Columbia Vac) that uses WiFi and connects through the Smart Life app. Is there a chance to add it to your adapter?

**Comment by Draghmar** [comment]
Going through the Setup process I've manage to add it as a switch which gave me an option to turn on/off. So if only there was more options to tinker with: changing mode and fan speed, it would make it 100% awesome! :)

**Comment by bewee** [comment]
Hi Draghmar,
That's totally possible, you just have to do the following in order to supply me with the necessary information:
In the adapter settings, change device type to (empty). Then make a screenshot of the device overview. The device's properties will no longer have names but only numbers. These are used to talk to the device. Try to mess around with these and describe what happens to the device whenever you enter different values there. (have a look at [this issue](https://github.com/Galveston01/tuya-adapter/issues/6#issuecomment-615447171) as an example)

**Comment by Galveston01** [comment]
Do you still experience this problem?

**Comment by bewee** [comment]
@Draghmar What's the status?

**Comment by Draghmar** [comment]
Yeah, sorry. I started testing but then I stumbled upon some issues with it. I tried to reset everything and lost connection in the process. I did the whole process of getting keys again a few times but it didn't work after the first one...Now I can't even connect using SmartHome app or Tuya even though WiFi connection is blinking on the device.
I did wrote to the producer but I didn't get response yet. At the moment I'm not sure if my unit is faulty or there's some issue with connection config or (and that is also quite possible) some issue with my WiFi - I have low quality router at the moment and I wouldn't be surprised if it was it's fault. I'm in the process of getting a new one which would eliminate this variable.
